Azure/sonic-buildimage
# [DX010] Remove executable permission of fancontrol.service for dx010

**- Why I did it**
Systemd keeps reporting WARNING in syslog
```
 WARNING systemd[1]: Configuration file /lib/systemd/system/fancontrol.service is marked executable. Please remove executable permission bits. Proceeding anyway.
```
This PR is to remove executable permission of fancontrol.service for dx010 platform.
**- How I did it**
Remove executable permission of ```/lib/systemd/system/fancontrol.service``` for dx010 platform.
**- How to verify it**
Verified on DX010, and confirm that the fancontrol service is running, and the WARNING log is no longer found in syslog.
```
admin@str-dx010-acs-4:~$ systemctl status fancontrol
● fancontrol.service - fan speed regulator
   Loaded: loaded (/lib/systemd/system/fancontrol.service; enabled; vendor preset: enabled)
   Active: active (running) since Mon 2020-10-12 02:33:26 UTC; 20min ago
 Main PID: 1303 (fancontrol.sh)
    Tasks: 3 (limit: 4671)
   Memory: 5.4M
   CGroup: /system.slice/fancontrol.service
           ├─1303 /bin/sh /etc/init.d/fancontrol.sh start
           ├─2460 /bin/bash /usr/local/bin/fancontrol /usr/share/sonic/device/x86_64-cel_seastone-r0/fancontrol-F2B
           └─4848 sleep 2
```
**- Which release branch to backport (provide reason below if selected)**




**- Description for the changelog**

Remove executable permission of fancontrol.service for dx010

**- A picture of a cute animal (not mandatory but encouraged)**
